packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -20,9 +20,15 @@ const GranularityOptions = {
   DATE: 'date',
 };
 
+const DATE_FORMAT = 'YYYY-MM-DD';
```

Comment:
shouldn't this format should be translated

Reply:
No it's for internal use only, it's the date only portion of an ISO-8601 date.

Reply:
Can we add an inline comment to document this use?